toolkit/utilitynetworks/src/main/java/com/arcgismaps/toolkit/utilitynetworks/ui/FeatureResultsDetailsScreen.kt

```diff
@@ -0,0 +1,137 @@
+
+/**
+ * Composable that displays the selected groups of feature results.
+ *
+ * @since 200.6.0
+ */
+@Composable
+internal fun FeatureResultsDetailsScreen(
+    selectedGroupName: String,
+    elementListWithSelectedGroupName: List<UtilityElement>,
+    onFeatureSelected: (UtilityElement) -> Unit,
+    onBackToNewTrace: () -> Unit,
+    onBackToResults: () -> Unit
+) {
+    BackHandler {
+        onBackToResults()
+    }
+    Surface(modifier = Modifier.fillMaxSize()) {
+        Column(modifier = Modifier
+            .fillMaxSize()
+            .padding(horizontal = 10.dp)) {
+
+            TabRow(onBackToNewTrace, 1)
+
+            Spacer(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .height(25.dp)
+            )
+
+            UpButton(stringResource(id = R.string.feature_results), onBackToResults)
+
+            Spacer(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .height(20.dp)
+            )
+
+            Row(
+                modifier = Modifier.fillMaxWidth(),
+                verticalAlignment = Alignment.CenterVertically,
+                horizontalArrangement = Arrangement.Center
+            ) {
+                Text(
+                    text = selectedGroupName,
+                    style = MaterialTheme.typography.titleLarge,
+                    maxLines = 1,
+                    overflow = TextOverflow.Ellipsis
+                )
+            }
+            LazyColumn {
+                item {
+                    FeatureList(elementListWithSelectedGroupName, onFeatureSelected)
+                }
+            }
+        }
+    }
+}
+
+@Composable
+private fun FeatureList(assetTypeList: List<UtilityElement>, onFeatureSelected: (UtilityElement) -> Unit) {
+    Surface(modifier = Modifier.fillMaxWidth()) {
+        Column {
+            ExpandableCardWithLabel(assetTypeList[0].assetType.name, value = assetTypeList.size.toString()) {
+                Column {
+                    assetTypeList.forEach { utilityElement ->
+                        HorizontalDivider()
+                        Row(
+                            modifier = Modifier
+                                .fillMaxWidth()
+                                .padding(start = 32.dp, end = 16.dp, top = 8.dp, bottom = 8.dp)
+                                .clickable { onFeatureSelected(utilityElement) },
+                        ) {
+                            Icon(
+                                imageVector = Icons.Sharp.ZoomInMap,
+                                contentDescription = stringResource(R.string.zoom_in),
+                                tint = MaterialTheme.colorScheme.primary
+                            )
+                            Text(
+                                modifier = Modifier.padding(start = 10.dp),
+                                text = "${stringResource(R.string.object_id)}: ${utilityElement.objectId}",
```

Comment:
this should use a stringResource format string so that it can be localized in RTL locales.
```suggestion
                                text = stringResource(R.string.object_id, utilityElement.objectId),
```